src/main/resources/regression/dl19-passage-jsonl.yaml

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
I don't think we need this since we have until tests?

Adding a regression means that I have to test it forever...

Reply:
Yes, it's for my sanity check. I can remove this file.

Reply:
yes please